internal/ansible/ansible.go

```diff
@@ -312,3 +308,40 @@ func (r *Runner) watch(
 		}
 	}
 }
+
+// filter the ansible job event based on a built-in schema of known necessary data
+func filterJobEvent(jobEventData []byte) []byte {
```

Comment:
Why do not use common pattern in Go here? I mean to return pointer and error. When calling of function is successful, then it return pointer and `nil`. In case of error `nil` and some error is returned. In this case it would be:

```go
// filterJobEvent filters the Ansible job event based on a built-in
// schema of known necessary data
func filterJobEvent(jobEventData []byte) ([]byte, error) {
	var filteredEvent PlaybookRunResponseMessageEventsElem
	err := json.Unmarshal(jobEventData, &filteredEvent)
	if err != nil {
		return nil, err
	}

	filteredData, err := json.Marshal(filteredEvent)
	if err != nil {
		return nil, err
	}
	return filteredData, nil
}
```

It simplifies the logic of the functions and it remove duplicity of log messages.